src/C/ogr_srs_api.jl

```diff
@@ -54,136 +60,96 @@ OGRSpatialReference destructor.
 ### Parameters
 * **hSRS**: the object to delete
 """
-function OSRDestroySpatialReference(arg1::OGRSpatialReferenceH)
+function OSRDestroySpatialReference(arg1)
     ccall((:OSRDestroySpatialReference, libgdal), Cvoid, (OGRSpatialReferenceH,), arg1)
 end
 
-
 """
     OSRReference(OGRSpatialReferenceH hSRS) -> int
 
 Increments the reference count by one.
 """
-function OSRReference(arg1::OGRSpatialReferenceH)
+function OSRReference(arg1)
     ccall((:OSRReference, libgdal), Cint, (OGRSpatialReferenceH,), arg1)
 end
 
-
 """
     OSRDereference(OGRSpatialReferenceH hSRS) -> int
 
 Decrements the reference count by one.
 """
-function OSRDereference(arg1::OGRSpatialReferenceH)
+function OSRDereference(arg1)
     ccall((:OSRDereference, libgdal), Cint, (OGRSpatialReferenceH,), arg1)
 end
 
-
 """
     OSRRelease(OGRSpatialReferenceH hSRS) -> void
 
 Decrements the reference count by one, and destroy if zero.
 """
-function OSRRelease(arg1::OGRSpatialReferenceH)
+function OSRRelease(arg1)
     ccall((:OSRRelease, libgdal), Cvoid, (OGRSpatialReferenceH,), arg1)
 end
 
-
 """
-    OSRValidate(OGRSpatialReferenceH) -> OGRErr
+    OSRValidate(OGRSpatialReferenceH hSRS) -> OGRErr
 
 Validate SRS tokens.
 """
-function OSRValidate(arg1::OGRSpatialReferenceH)
+function OSRValidate(arg1)
     ccall((:OSRValidate, libgdal), OGRErr, (OGRSpatialReferenceH,), arg1)
 end
 
-
-"""
-    OSRFixupOrdering(OGRSpatialReferenceH hSRS) -> OGRErr
-
-Correct parameter ordering to match CT Specification.
-"""
-function OSRFixupOrdering(arg1::OGRSpatialReferenceH)
-    ccall((:OSRFixupOrdering, libgdal), OGRErr, (OGRSpatialReferenceH,), arg1)
-end
-
-
-"""
-    OSRFixup(OGRSpatialReferenceH hSRS) -> OGRErr
-
-Fixup as needed.
-"""
-function OSRFixup(arg1::OGRSpatialReferenceH)
-    ccall((:OSRFixup, libgdal), OGRErr, (OGRSpatialReferenceH,), arg1)
-end
-
-
-"""
-    OSRStripCTParms(OGRSpatialReferenceH hSRS) -> OGRErr
-
-Strip OGC CT Parameters.
-"""
-function OSRStripCTParms(arg1::OGRSpatialReferenceH)
-    ccall((:OSRStripCTParms, libgdal), OGRErr, (OGRSpatialReferenceH,), arg1)
-end
-
-
 """
     OSRImportFromEPSG(OGRSpatialReferenceH hSRS,
                       int nCode) -> OGRErr
 
 Initialize SRS based on EPSG GCS or PCS code.
 """
-function OSRImportFromEPSG(arg1::OGRSpatialReferenceH, arg2::Cint)
+function OSRImportFromEPSG(arg1, arg2)
     ccall((:OSRImportFromEPSG, libgdal), OGRErr, (OGRSpatialReferenceH, Cint), arg1, arg2)
 end
 
-
 """
     OSRImportFromEPSGA(OGRSpatialReferenceH hSRS,
                        int nCode) -> OGRErr
 
-Initialize SRS based on EPSG GCS or PCS code.
+Initialize SRS based on EPSG CRS code.
```

Comment:
Maybe we should have a way of storing the results (function names and their docstrings), so that we can write tools in the future to generate a CHANGELOG from run to run. 😝

Reply:
Yeah this PR is unfortunately a conflation of a new GDAL and changes to the docstring generation.

It's quite easy to write all docstrings to JSON or whatever, was actually doing that earlier for this PR, but since it introduced an extra step I took it out.

Reply:
yeah we don't have to do that for this PR; was just thinking out loud